WorkloadManagementSystem/Utilities/PilotCStoJSONSynchronizer.py

```diff
@@ -161,13 +161,17 @@ def _getCSDict(self):
 
         for ce in ceList['Value']:
           ceType = gConfig.getValue('/Resources/Sites/' + grid + '/' + site + '/CEs/' + ce + '/CEType')
+          localCEType = gConfig.getValue('/Resources/Sites/' + grid + '/' + site + '/CEs/' + ce + '/LocalCEType')
 
           if ceType is None:
             # Skip but log it
             self.log.error('CE has no option CEType! - skipping', ce + ' at ' + site)
           else:
             pilotDict['CEs'][ce] = {'Site': site, 'GridCEType': ceType}
 
+          if localCEType is not None:
+            pilotDict['CEs'][ce] = {'Site': site, 'LocalCEType': localCEType}
```

Comment:
As this replaces the dictionary in, it overwrites the 'GridCEType' entry that is present  if 'ceType' is set. 
Is that intentional that either GridCEType or LocalCEType is set?

Reply:
Ah, good catch. Fixed.